src/AspNet.Security.OAuth.EVEOnline/EVEOnlineAuthenticationDefaults.cs

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using Microsoft.AspNetCore.Builder;
+
+namespace AspNet.Security.OAuth.EVEOnline {
+    /// <summary>
+    /// Default values used by the EVEOnline authentication middleware.
+    /// </summary>
+    public static class EVEOnlineAuthenticationDefaults {
+        /// <summary>
+        /// Default value for <see cref="AuthenticationOptions.AuthenticationScheme"/>.
+        /// </summary>
+        public const string AuthenticationScheme = "EVEOnline";
+
+        /// <summary>
+        /// Default value for <see cref="RemoteAuthenticationOptions.DisplayName"/>.
+        /// </summary>
+        public const string DisplayName = "EVEOnline";
+
+        /// <summary>
+        /// Default value for <see cref="AuthenticationOptions.ClaimsIssuer"/>.
+        /// </summary>
+        public const string Issuer = "EVEOnline";
+
+        /// <summary>
+        /// Default value for <see cref="RemoteAuthenticationOptions.CallbackPath"/>.
+        /// </summary>
+        public const string CallbackPath = "/signin-eveonline";
+
+        public static class Tranquility
```

Comment:
Yes, Tranquility is live server and Singularity is test server, both support OAuth, usually new features that require oauth token needs to be tested on Singularity, that's why I decided to also add an option to use it. Live server is default option.